spec/features/officer/view_landing_applications_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# Feature: Officer views list of landing applications
+#   So that I can identify and prioritise my work assessing landing applications
+#   As a DfSSETA officer
+#   I want to see a list of landing applications
+
+RSpec.feature "Officer views list of landing applications" do
+  # Scenario: Officer views list of landing applications
+  #   Given there are existing applications in the database
+  #   When I go to the landing applications page
+  #   Then I should see a list of applications
+
+  before do
+    create_applications
+  end
+
+  scenario "Officer views list of landing applications" do
+    visit officer_landing_applications_path
+    should_see_list_of_applications
+    should_see_applications_ordered_by_date_submitted
+  end
+
+  def should_see_applications_ordered_by_date_submitted
+    expect(
+      find_all("#submission-date").map { |ele| ele.text }
+    ).to eq(
+      ["2024-01-01",
+        "2023-01-01",
+        "2022-01-01"]
+    )
+  end
+
+  def should_see_list_of_applications
+    landable_bodies = LandableBody.all
```

Comment:
I think `landable_bodies` is now redundant?

Reply:
ah yes. I keep forgetting to run `rake standard`!